### ISSUE TITLE
Disconnecting Android Auto Does Not Save Playback Position

### ISSUE THREAD

**Issue by timjtech** [advplyr/audiobookshelf-app#402]
### Steps to reproduce

1. Start Vehicle, open ABS on Android Auto
2. Playback for some time
3. Turn off the vehicle and remove the phone during playback
4. Restart the vehicle and connect to Android Auto
5. Begin playback

### Expected behaviour
- Tell us what should happen
- Playback resumes at last positon

### Actual behaviour
- Tell us what happens
- Playback resumes from the incorrect position

### Environment data

Audiobookshelf Version:

- [ x ] Android App?
- [ ] iOS App?

#### Android Issue
Android version: v0.9.57-beta
Android Auto v8.2.626444

Device model:
Samsung S21 International

Stock or customized system:
Stock (unmodified)

#### iOS Issue
iOS Version:

iPhone model:

**Comment by advplyr** [comment]
Is this an audiobook or podcast episode? Is it downloaded or are you streaming from your server?

**Comment by timjtech** [comment]
This is a downloaded audiobook

**Comment by advplyr** [comment]
When you are listening to the downloaded book are you still connected to your server?

Are you able to test this on a book you are streaming or are you only connecting to your server locally?

**Comment by timjtech** [comment]
Connection would be intermittent over the internet; I drove through several areas with no cell phone connectivity. The app is connected to my local sever over the internet. I downloaded the book for a long trip with intermittent connections.

Please suggest parameters for testing this while streaming?

**Comment by advplyr** [comment]
If you repeat the steps you mentioned above but just listen to a book you haven't downloaded yet instead of a downloaded book.

**Comment by timjtech** [comment]
As I don't have a place without connectivity I turned on Airplane mode while listening to a streaming book. After listening for 2 minutes on Airplane mode I turned off the vehicle and removed the device. When I reattached the device and disabled airplane mode my progress was lost.

**Comment by Hackmodford** [comment]
I think I experienced something similar on iOS.

I was playing a downloaded book. I don’t believe I was connected to my server as I was in my car. When I got to my garage I assume bluetooth disconnecting paused the playback. Hours later I opened the app to resume. I could tell it was a fresh start (I assume iOS killed it at some point). The playback wanted to resume from the position I was at when I left.
